packages/nexus/src/workflow-helpers.ts

```diff
@@ -0,0 +1,153 @@
+import { convertNexusLinkToWorkflowEventLink, convertWorkflowEventLinkToNexusLink } from './link-converter';
+import { getClient, getHandlerContext, log } from './context';
+
+declare const isNexusWorkflowHandle: unique symbol;
+
+/**
+ * A handle to a running workflow that is returned by the {@link startWorkflow} helper.
+ * This handle should be returned by {@link WorkflowRunOperationStartHandler} implementations.
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export interface WorkflowHandle<_T> {
+  readonly workflowId: string;
+  readonly runId: string;
+
+  /**
+   * Virtual type brand to maintain a distinction between {@link WorkflowHandle} provided by the
+   * {@link startWorkflow} helper (which will have attached links, request ID, completion URL, etc)
+   * and the `WorkflowHandle` type returned by the {@link WorkflowClient.start}.
+   *
+   * @internal
+   * @hidden
+   *
+   * @experimental Nexus support in Temporal SDK is experimental.
+   */
+  readonly [isNexusWorkflowHandle]: typeof isNexusWorkflowHandle;
+}
+
+/**
+ * Options for starting a workflow using {@link startWorkflow}, this type is identical to the
+ * client's `WorkflowStartOptions` with the exception that `taskQueue` is optional and defaults
+ * to the current worker's task queue.
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type WorkflowStartOptions<T extends Workflow> = Replace<ClientWorkflowStartOptions<T>, { taskQueue?: string }>;
+
+/**
+ * Starts a workflow run for a {@link WorkflowRunOperationStartHandler}, linking the execution chain
+ * to a Nexus Operation (subsequent runs started from continue-as-new and retries). Automatically
+ * propagates the callback, request ID, and back and forward links from the Nexus options to the
+ * Workflow.
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export async function startWorkflow<T extends Workflow>(
+  ctx: nexus.StartOperationContext,
+  workflowTypeOrFunc: string | T,
+  workflowOptions: WorkflowStartOptions<T>
+): Promise<WorkflowHandle<T>> {
+  const { client, taskQueue } = getHandlerContext();
+  const links = Array<temporal.api.common.v1.ILink>();
+  if (ctx.inboundLinks?.length > 0) {
+    for (const l of ctx.inboundLinks) {
+      try {
+        links.push({
+          workflowEvent: convertNexusLinkToWorkflowEventLink(l),
+        });
+      } catch (error) {
+        log.warn('failed to convert Nexus link to Workflow event link', { error });
+      }
+    }
+  }
+  const internalOptions: InternalWorkflowStartOptions[typeof InternalWorkflowStartOptionsSymbol] = {
+    links,
+    requestId: ctx.requestId,
+  };
+
+  if (workflowOptions.workflowIdConflictPolicy === 'USE_EXISTING') {
+    internalOptions.onConflictOptions = {
+      attachLinks: true,
+      attachCompletionCallbacks: true,
+      attachRequestId: true,
+    };
+  }
```

Comment:
Minor, but Go and Java set these all to `true` unconditionally:

Go: https://github.com/temporalio/sdk-go/blob/9ca81c60/temporalnexus/operation.go#L332
Java: https://github.com/temporalio/sdk-java/blob/b6b42903/temporal-sdk/src/main/java/io/temporal/internal/common/InternalUtils.java#L153-L158

The server honors the options conditional on the conflict policy: https://github.com/temporalio/temporal/blob/fdb7f31a4/service/history/api/startworkflow/api.go#L423-L425

I suggest that all SDKs follow Go and Java and pass them unconditionally, and allow the server to decide what goes into effect. This will reduce the possibility of differences creeping in if we add/change conflict policies in the future.